dali/operators/image/crop/bbox_crop.cc

```diff
@@ -529,20 +546,21 @@ class RandomBBoxCropImpl : public OpImplBase<CPUBackend> {
     bool success = false;
     Box<ndim, float> crop{};
     std::vector<Box<ndim, float>> boxes;
-    std::vector<int> labels;
+    SmallVector<int, 128> bbox_indices;
```

Comment:
I think that's a bit too big an object - perhaps it should be a regular vector instead or at the very least we should be much more conservative wrt static size (8-16, not 128).